community/server/src/main/coffeescript/ribcage/ui/LineChartTimeTicker.coffee

```diff
@@ -28,63 +28,54 @@ define(
     ###
     class LineChartTimeTicker
       
-      timezoneOffset : new Date().getTimezoneOffset() * 60
-      
       scales : [
         {  # 30-min span
            maxSpan : 31 * 60, 
            tickLength : 5 * 60, 
            dateFormat : "HH:MM",
-           findFirstTickFrom : (startTime) -> 
-             timezoneOffset = new Date().getTimezoneOffset()
-             time = new Date((startTime+timezoneOffset*60) * 1000)
-             new Date(time.getFullYear(), time.getMonth(), time.getDate(), 0,timezoneOffset,0).getTime() / 1000
+           findFirstTickFrom : (startTime) ->
```

Comment:
Hi @cleishm, while you're here, can't you extract the repeated code into a variable and reuse it?